string-manipulation-1/src/is-lower-cased.js

```diff
@@ -0,0 +1,25 @@
+/* exported isLowerCased */
+
+/**
+ * PSEUDOCODE:
+ * Define a function called 'isLowerCased' that takes in one parameter 'word'.
+ *  Declare variable 'lowerCount' and set initial value to 0.
+ *  Begin a loop that iterates through all characters in 'word' where
+ *    For each index in the string:
+ *      If the character is uppercase, increment 'lowerCount'.
+ *  Return if lowerCount matches length of 'word'.
+ **/
+
+/**
+ * CODE IMPLEMENTATION:
+ */
+
+function isLowerCased(word) {
+  var lowerCount = 0;
+  for (var i = 0; i < word.length; i++) {
+    if (word[i] === word[i].toLowerCase()) {
+      lowerCount++;
+    }
+  }
+  return lowerCount === word.length;
```

Comment:
Instead of looping, you can just turn `word` into all lowercase, and compare it to the original value! Here's an example
```
function isLowerCased(word) {
  return word === word.toLowerCase();
}
```